imports/core/ui/components/AppHeader.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <header>
     <div class="container">
-      <img class="logo" src="/images/logo_white.png" alt="logo" />
+      <router-link to="/" class="logo" src="/images/logo_white.png" alt="logo" tag="img"/>
```

Comment:
It's easy to add a css line `cursor: pointer;` do not need to make it complicated bro.

Reply:
`a` tag is specially handled by browser: path is displayed at the bottom, you can copy path or open it in a new tab. It is more semantic than a custom tag with JS behavior.
I think we also should use `<a>` in filters and in proposal list.

Reply:
Sorry to say, native `<a>` tag causes reloading and resetting app states, that why I use `<router-link>` for filters, it's also rendering `<a>` tag with js observation attached. I need to use router `push` for proposal item because I have to trigger another mutation `agreeOrDoubtProposal` with a boolean payload to make sure if we click on upvote or downvote, we get corresponding pre-selected choice on single proposal page. Do you have any idea ?

Reply:
Yes, I mean `<router-link tag="a">` when spoke about `<a>`. Firstly I want to prove that it is better for the logotype.
In `ProposalItem` we can put the pre-selected choice in the URL or use 68f2c6f220747100e37c5d290802e52c42cc81c8.